src/lib.rs

```diff
@@ -11,6 +11,60 @@
 
 //! Library for serializing the RSS web content syndication format.
 //!
+//! # Building
+//!
+//! A channel can be built using the Builder functions.
+//!
+//! ## Example
+//!
+//! ```
+//! extern crate rss;
+//!
+//! use rss::{ChannelBuilder, ImageBuilder};
+//!
+//! fn main()
+//! {
+//!     let image = ImageBuilder::new()
+//!             .url("http://jupiterbroadcasting.com/images/LAS-300-Badge.jpg")
+//!             .title("LAS 300 Logo")
+//!             .link("http://www.jupiterbroadcasting.com")
+//!             .width(Some(88))
+//!             .height(Some(88))
+//!             .description(Some("This is a test".to_string()))
+//!             .finalize()
+//!             .unwrap();
+//!
+//!     let description = "Ogg Vorbis audio versions of The Linux ".to_string()
+//!     + "Action Show! A show that covers everything geeks care about in the "
+//!     + "computer industry. Get a solid dose of Linux, gadgets, news events "
+//!     + "and much more!";
+//!
+//!     let channel = ChannelBuilder::new()
+//!             .title("The Linux Action Show! OGG")
+//!             .link("http://www.jupiterbroadcasting.com")
+//!             .description(description.as_ref())
+//!             .language(None)
+//!             .copyright(None)
+//!             .managing_editor(None)
+//!             .webmaster(None)
```

Comment:
For the sake of brevity, seems like we could remove a lot of these lines where we're specifying `None` since that's the default value? Not a big deal though if you prefer to leave these lines in